.gitignore

```diff
@@ -11,7 +11,7 @@ log.txt
 
 # only include appsettings for a few default environments
 appsettings.*.json
-dab-config.*.json
+dab-config*.json
 *.dab-config.json
```

Comment:
will this not exclude the config files from the test project? Is this intended/why?

Reply:
Interesting, because even the original regex should exclude the config files in test project.

Reply:
actually this change is not required. thanks for pointing out

Reply:
did you determine otherwise: that the change is required? I see:
```
dab-config.*.json
dab-config.json
```

Reply:
yeah, while testing changes i was creating dab-config.json and it was getting tracked as well, which should not be the case. hence this change.